src/content/docs/infrastructure/install-infrastructure-agent/linux-installation/enable-logs-monitored-containers.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: Enable logs for instrumented containers
+tags:
+  - Infrastructure
+  - Install the infrastructure agent
+  - Linux installation
+metaDescription: New Relic's infrastructure monitoring agent automatically instruments container runtimes to collect container metrics and metadata.
+freshnessValidatedDate: never
+---
+
+import infrastructureAddNamesLogs from 'images/infrastructure_screenshot-crop_add-names-logs.webp'
+
+import infrastructureAddContainersLogPath from 'images/infrastructure_screenshot-crop_add-containers-log-path.webp'
+
+import logsParsingUi from 'images/infrastructure_screenshot-full_logs-parsing-ui.webp'
+
+Our [infrastructure agent](/docs/infrastructure/infrastructure-monitoring/get-started/get-started-infrastructure-monitoring) can gather logs from your containers. When you enable logs from the infrastructure agent, the agent will automatically assign those logs to the hosts running Docker.
+
+## Requirements [#requirements]
+
+To collect logs from your containers, you need to have:
+
+    * Infrastructure agent [1.8.32](/docs/release-notes/infrastructure-release-notes/infrastructure-agent-release-notes/new-relic-infrastructure-agent-1832) or higher running on Linux
+    * CentOS version 6.0 or higher, if you're using CentOS
+    * Docker with engine from [v1.12](https://docs.docker.com/engine/release-notes/prior-releases/#1120-2016-07-28) or other [`containerd`-based container runtimes](https://kubernetes.io/docs/setup/production-environment/container-runtimes)
+
+## Collect logs from your containers [#collect-logs]
+
+1. Open the configuration file for the infrastructure agent and add this snippet:
+
+    ```yml
+    - name: containers
+      file: /var/lib/docker/containers/*/*.log
+    ```
```

Comment:
```suggestion
1. Create a file under `/etc/newrelic-infra/logging.d/docker-containers.yml` with the following contents:

```yml
logs:
    - name: docker_containers
      file: /var/lib/docker/containers/*/*.log
```
```

Reply:
Added to the PR!